.github/workflows-source/schedule-daily.yml

```diff
@@ -146,13 +146,18 @@ jobs:
     steps:
       - <<: *checkout
       - <<: *before-script
+      - name: Set Benchmark Targets
+        shell: bash
+        run: |
+          set -xeuo pipefail
+          echo "BENCHMARK_TARGETS=$(bazel query 'attr(tags, system_test_benchmark, //rs/...)')" >> $GITHUB_ENV
```

Comment:
If we end up having no system tests with this tag, this CI job will probably break.

Reply:
wouldn't that be good s.t. we notice that something is off? if there are no tests, we should either delete it or adjust, right?

Reply:
That's a good reasoning. :+1: